README.md

```diff
@@ -52,6 +52,17 @@ almost always be binary compatible with prior minor releases from the same major
 Patch 4.x.y increments (such as 4.0.0 -> 4.0.1, 4.1.1 -> 4.1.2, etc) will occur for bug fixes only and will always be binary compatible
 with prior patch releases of the same minor release branch.
 
+#### @Alpha
```

Comment:
I agree there's a degree of redundancy. However, users could refer to the README for quick, high-level insights, and noting that there are might be APIs marked as Alpha or Beta could prompt them to pay attention to the Javadocs. Additionally, making contributors aware of the parts of the API that are still under development might encourage their involvement. Since we rarely introduce new annotations like this, i believe the redundancy should not be problematic.